_component_design/help-beacon.md

```diff
@@ -4,13 +4,12 @@ parent: Help
 secondary: Notifications
 layout: component
 category: Components
-usage: |
-  These tooltips provide users with additional information while completing
-  tasks and alert them to new features.
-preview-image: components/preview-help.svg
-status: deprecated, in-progress
+usage: >
+  The Beacon can be used to identify the location of products or features in the control panel. It can also be used to indicate advanced functionality to existing features
+preview-image: preview-images/help-beacon.svg
+status: in-progress
 resource: true
-last-modified: 2017-08-17
+last-modified: 2017-10-04
```

Comment:
Small nit: I'd leave the last mod'd date alone unless you're adjusting the content of the page itself. If we change this one here, we probably need to change each date on each file touched by this PR.

Reply:
Reverted in commit fe6a7cf and added to this PR.